website/src/app/kb/user-guides/windows-client/readme.mdx

```diff
@@ -1,6 +1,6 @@
 # Windows Client
 
-Firezone supports Microsoft Windows with a native client for the Intel x86_64
+Firezone supports Microsoft Windows with a native client for the Intel/AMD x86_64
```

Comment:
I added "AMD" here because x86_64 was originally amd64, and then Intel also adopted it